default/python/universe_generation/monsters.py

```diff
@@ -131,7 +132,8 @@ def generate_monsters(monster_freq, _systems, home_systems):
     systems = [s for s in _systems if s not in empire_exclusions]
 
     # Fleet plans that include ships capable of altering starlanes.
-    ## @content_tag{CAN_ALTER_STARLANES} universe_generator special handling for fleets containing a hull design with this tag.
+    # @content_tag{CAN_ALTER_STARLANES} universe_generator special handling for
+    # fleets containing a hull design with this tag.
```

Comment:
For special command processing, doxygen requires `##`. [doxygen pythonblocks](http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html#pythonblocks) / [cppapi doc for content_tag alias](http://freeorion.github.io/cppapi/master/content_tags_alias.html)

The param/content for `content_tag` alias is limited to the first line break.